nncf/common/hardware/config.py

```diff
@@ -11,30 +11,25 @@
  limitations under the License.
 """
 
-from collections import OrderedDict
-from enum import Enum
-from pathlib import Path
-from typing import Any
-from typing import Dict
-from typing import List
-from typing import Optional
-from typing import Set
-from typing import Type
-
 import addict as ad
 import jstyleson as json
```

Comment:
Do we need to add jstyleson to https://github.com/openvinotoolkit/nncf/blob/develop/beta/requirements.txt?

Reply:
Fixed in the https://github.com/openvinotoolkit/nncf/pull/729